### ISSUE TITLE
账号显示登录之后会闪退

### ISSUE THREAD

**Issue by keepmi** [Yudaotor/EsportsHelper#81]
这三天从2.3.0到2.3.1都会出现这种情况

**Comment by Yudaotor** [comment]
因为Riot最近在登录的时候加了一个概率出现的图片验证码,所以会报错从而闪退.目前解决方案为:
1. 提前使用浏览器登录观赛网站,并且记住密码,然后使用userDataDir配置来进行自动登录,详细可见readme和wiki.
2. 不是每次登录都会出现这个问题,所以可能过段时间就可以了.
3. 不使用无头模式即headless模式,在登录时出现图片验证码的时候手动验证,从而正常登录.

**Comment by keepmi** [comment]
> 因为Riot最近在登录的时候加了一个概率出现的图片验证码,所以会报错从而闪退.目前解决方案为:
> 
> 1. 提前使用浏览器登录观赛网站,并且记住密码,然后使用userDataDir配置来进行自动登录,详细可见readme和wiki.
> 2. 不是每次登录都会出现这个问题,所以可能过段时间就可以了.
> 3. 不使用无头模式即headless模式,在登录时出现图片验证码的时候手动验证,从而正常登录.

好的

**Comment by Gimmeday** [comment]
我这里总是会出现图片验证码需求，并不是概率性出现，并且只是在chrome上会如此，哪怕是手动使用chrome登录也是如此，而当我使用其他浏览器比如edge登录时他总是正常的。

**Comment by redr00t** [comment]
> any solution for the docker user? @redr00t

I don't think so, using Docker it is not possible to interact with the graphical interface of the web browser, headless mode is mandatory.
the only solution I can think of is to provide the ability to mount (using the docker-compose.yml file) the user data folder.
I'll try to do it tomorrow.

**Comment by 0xAB51NTH** [comment]
> 1. Login to the site in advance with your browser and remember your password, then use the userDataDir configuration for automatic login, see the readme and wiki for details.

If someone wants to be able to use chrome browser and run the bot just copy the folder (user data) into any place and point the configuration file to it, because when chrome is turned on it will prevent esport bot from reading the cookies file (atleast from my testing, also idk if the bot needs something more than cookies but if its the only thing then you just need the default folder inside "user data", the structure of "user data/default" has to stay tho).

**Comment by redr00t** [comment]
> > > any solution for the docker user? @redr00t
> > 
> > 
> > I don't think so, using Docker it is not possible to interact with the graphical interface of the web browser, headless mode is mandatory. the only solution I can think of is to provide the ability to mount (using the docker-compose.yml file) the user data folder. I'll try to do it tomorrow.
> 
> any possibility to link VNC so we can pass the captcha manually then turn on headless mode?

https://github.com/Yudaotor/EsportsHelper/pull/95
